api/utils/validations.js

```diff
@@ -0,0 +1,87 @@
+const { check, validationResult } = require('express-validator');
+
+exports.checkAddProduct =
+    [
+        check('title')
+            .trim()
+            .isLength({ min: 3 })
+            .withMessage('Minimum 3 characters required')
```

Comment:
You can create a file error-messages.js where you will add all messages that are for validations and so on. And remove magic values from here.